ironfish-rust-nodejs/index.d.ts

```diff
@@ -3,12 +3,6 @@
 
 /* auto-generated by NAPI-RS */
 
-export class ExternalObject<T> {
-  readonly '': {
-    readonly '': unique symbol
-    [K: symbol]: T
-  }
-}
```

Comment:
Not sure why the build re-generated this, can we revert this? I'll ask rohan why this is being modified? Maybe this is a rust version thing?

Reply:
Can revert, will leave it as-is til tomorrow